README.adoc

```diff
@@ -142,6 +142,40 @@ NOTE: embedded.kafka.enabled=false will cause DockerNotPresentException if you d
 |Memsql will start
 |===
 
+=== Other specific container related properties
+|===
+|Setting name | Default value |Description
+
+|embedded.{module-name}.waitTimeoutInSeconds
+|60
+|Waiting time for a container to start in seconds
+
+|embedded.{module-name}.enabled
+|true
+|Enables a container to be started on startup
+
+|embedded.{module-name}.reuseContainer
+|false
+|Enables a reuse container testcontainers feature.
+
+|embedded.{module-name}.command
+|null
+|List of keywords which combines into command for container startup. Some modules ship container's commands by default, so resetting this value may lead to incorrect work of container.
+
+|embedded.{module-name}.env
+|null
+|key-value map of additional environment variables. Where key is name of variable and value is actual value of it.
+
+|embedded.{module-name}.filesToInclude
+| null
+a|List of files to include objects.
+Each object should have two parameters:
```

Comment:
Could you please add an example for this one?

Reply:
added, plus added test for utils method